pyecharts/engine.py

```diff
@@ -85,6 +80,12 @@ def echarts_container(env, chart):
         _container_and_text += EXTRA_TEXT_FORMATTER.format(
             text=_text, style=_style
         )
+
+    _container_and_text += CHART_DIV_FORMATTER.format(
```

Comment:
Please update the comment of this function, which now returns `<p></p><div></div>` instead of a `div` element.